packages/amplify_authenticator/lib/src/widgets/form.dart

```diff
@@ -174,27 +174,33 @@ class AuthenticatorFormState<T extends AuthenticatorForm>
     if (widget.child != null) {
       return Form(
         key: formKey,
-        child: widget.child!,
+        child: AutofillGroup(
+          onDisposeAction: AutofillContextAction.commit,
+          child: widget.child!,
+        ),
       );
     }
 
     final runtimeActions = this.runtimeActions(context);
     return Form(
       key: formKey,
-      child: Column(
-        children: [
-          ...allFields,
-          Column(
-            children: [
-              if (widget.actions.isNotEmpty) const SizedBox(height: 4),
-              ...widget.actions,
-              if (runtimeActions.isNotEmpty) ...[
-                const Divider(),
-                ...runtimeActions,
-              ]
-            ].spacedBy(const SizedBox(height: 12)),
-          ),
-        ],
+      child: AutofillGroup(
```

Comment:
I am not that familiar with auto fill groups, but I don't think this is how `AutofillGroup` should be used. My understanding is that it should be used when all the fields are related. For example, a multi field address, a multi field credit card input, or possibly a username/password group. The use of `AutofillGroup` around the entire form seems to lead to some odd behavior, at least on iOS.

I noted a change above that will have to be made to get auto fill working for phone number as a sign up alias. When the change noted above is made, the sign up form has some strange behavior. It suggests using an email for the phone number field, a username for the password, etc. Removing this `AutofillGroup` resolves that (phone shows a phone number, password shows a password, etc.).

I'm not sure if removing it all together is the correct solution. It might provide some utility. I'll have to look into it a bit more. If this is something you are familiar with, please feel free to provide some insight.

Reply:
I have reviewed the official documentation and, as you mention, my use of `AutoFillGroup` was not correct. I've removed it.

It might be useful to update the code in the future to group the fields automatically into logical `AutoFillGroup`s

Reply:
Did you test the autofill functionality after removing AutoFillGroup with email and password?
From what I've tested (iOS device), autofilling email/password or auto-generating a password via keychain **only** works with AutofillGroup. However, telephoneNumber an givenName works without AutoFillGroup.